config/default.schema.json

```diff
@@ -90,6 +90,11 @@
 			"default": false,
 			"description": "JWTs that are not in the whitelist are added to it, instead of invalidated. To be used for the first 2 hours after release."
 		},
+		"SC_TIME_ZONE": {
```

Comment:
We don't need this environment variable any more, because all dates are saved in UTC format on server

Reply:
done ;)